packages/dates/src/get-date-time-parts.ts

```diff
@@ -57,7 +57,7 @@ function getWeekdayValue(weekday: string) {
   return weekdays[weekday];
 }
 
-// eslint-disable-next-line shopify/no-fully-static-classes
+// eslint-disable-next-line shopify/no-fully-static-classes, @typescript-eslint/no-extraneous-class
```

Comment:
I think we should just 🔥 our rule and go with the typescript one. It checks more than just fully static classes but also classes that are just a constructor or completely empty.

Reply:
For the TypeScript config, sure, it's still good for the JS config.